### DISCUSSION TITLE
Stop IM converting to image to grayscale

### DISCUSSION THREAD

**Discussion by ghost** [ImageMagick/ImageMagick#6074] [Help] [answered]
### ImageMagick version

7.1.0-62-Q16

### Operating system, version and so on

Windows 10

### Description

Hello,

The following command converts the image to grayscale:

magick input.tiff -fill white -colorize 100%% output.tiff

I suppose this is expected, but how do I prevent this because I need it to remain a rgb color image?

Any ideas?

**Comment by fmw42** [comment, marked as answer]
try

`magick input.tiff -fill white -colorize 100%% -type truecolor output.tiff`

or

`magick input.tiff -fill white -colorize 100%% -colorspace sRGB output.tiff`

**Reply by ghost** [reply]
Yup that worked! Thank you.
